### ISSUE TITLE
Exasol L026 - Fails on SELECT INTO TABLE statement

### ISSUE THREAD

**Issue by sti0** [sqlfluff/sqlfluff#1830]
In Exasol we could insert data into a table like this:

```
select * into table schm.tab from schm.tab2;
```

## Expected Behaviour
Rule L026 should pass

## Observed Behaviour
Rule was raised
```
L:   2 | P:  21 | L026 | Reference 'schm.tab' refers to table/view not found in
                       | the FROM clause or found in parent subquery
```

## Steps to Reproduce
Lint the statement above

## Dialect
exasol

## Version
Include the output of `sqlfluff --version` along with your Python version
sqlfluff 0.7.1
python 3.8.12
## Configuration
default

**Comment by tunetheweb** [comment]
This is a great first issue for anyone looking to contribute to SQLFluff: https://github.com/sqlfluff/sqlfluff/wiki/Contributing-Dialect-Changes

**Comment by barrywhart** [comment]
As noted by @jpy-git, this is a similar issue as #2265. There's an open PR for that issue (#2336). If we don't address it in that PR, it should be a pretty straightforward enhancement to that fix.
